src/src/com/microsoft/aad/adal/CacheKey.java

```diff
@@ -94,7 +103,10 @@ public static String createCacheKey(String authority, String resource, String cl
             key.mAuthority = (String)key.mAuthority.subSequence(0, key.mAuthority.length() - 1);
         }
 
-        key.mClientId = clientId.toLowerCase(Locale.US);
+        if (clientId != null) {
```

Comment:
Cachekey will store family client id. 

---
In reply to: [62928372](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/611#discussion_r62928372) [](ancestors = 62928372)